xwiki-commons-core/xwiki-commons-extension/xwiki-commons-extension-api/src/main/java/org/xwiki/extension/job/history/internal/ExtensionJobHistoryRecorder.java

```diff
@@ -46,7 +46,7 @@
 import org.xwiki.job.event.JobStartedEvent;
 import org.xwiki.job.event.status.JobStatus;
 import org.xwiki.job.event.status.QuestionAnsweredEvent;
-import org.xwiki.job.internal.AbstractJobStatus;
+import org.xwiki.job.AbstractJobStatus;
```

Comment:
This impacts a condition in `#onJobStarted` which is called from `#onEvent` hence the legacy class I added to support the condition with the old class.